webpack.config.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path');
+const nodePath = process.env.NODE_PATH || './node_modules';
+
+module.exports = {
+    mode: "production",
+    entry: path.resolve(__dirname, './kuma/javascript/src/index.jsx'),
+    output: {
+        filename: 'react.js',
+        path: path.resolve(__dirname, './kuma/javascript/dist/')
+    },
+    module: {
+        rules: [{
+            "test": /\.(js|jsx)$/,
```

Comment:
We could change this to the following:

```suggestion
            test: /\.jsx?$/,
```

Reply:
Thanks, Schalk. I *thought* I'd put the babel config in package.json. I wonder if I lost it while fixing a merge confict or something?  My intent is to be using that emotion plugin. It also gives us class names that have meaninful names instead of completely random strings, which will be nice.

Reply:
Ah. The babel config is inside the webpack config. I've added the pretter preset, and am going to keep babel inside webpack, unless we come up with some reason to separate them.